python/lsst/ctrl/bps/htcondor/htcondor_service.py

```diff
@@ -1976,13 +1998,29 @@ def _create_periodic_release_expr(memory, multiplier, limit):
     # the entire expression should evaluate to FALSE when the job is not HELD.
     # According to ClassAd evaluation semantics FALSE && UNDEFINED is FALSE,
     # but better safe than sorry.
-    was_mem_exceeded = (
-        "JobStatus == 5 "
-        "&& (HoldReasonCode =?= 34 && HoldReasonSubCode =?= 0 "
-        "|| HoldReasonCode =?= 3 && HoldReasonSubCode =?= 34)"
-    )
+    is_held = "JobStatus == 5"
+    is_retry_allowed = "NumJobStarts <= JobMaxRetries"
+
+    memory_expr = ""
```

Comment:
Can we please rename `memory_expr` to `mem_expr` to keep it consistent with the naming convention used in `_create_periodic_remove_expr()` and other auxiliary variables in this function?